js/compiler.ts

```diff
@@ -541,6 +637,50 @@ window.compilerMain = function compilerMain() {
   };
 };
 
+const decoder = new TextDecoder();
+
+function getCompilerConfig(
+  compilerType: string
+): { path: string; data: string } {
+  const builder = flatbuffers.createBuilder();
+  const compilerType_ = builder.createString(compilerType);
+  msg.CompilerConfig.startCompilerConfig(builder);
+  msg.CompilerConfig.addCompilerType(builder, compilerType_);
+  const inner = msg.CompilerConfig.endCompilerConfig(builder);
+  const baseRes = sendSync(builder, msg.Any.CompilerConfig, inner);
+  assert(baseRes != null);
+  assert(msg.Any.CompilerConfigRes === baseRes!.innerType());
+  const res = new msg.CompilerConfigRes();
+  assert(baseRes!.inner(res) != null);
+
+  const path = res.path()!;
+  assert(path != null);
+  const dataArray = res.dataArray()!;
+  assert(dataArray != null);
+  const data = decoder.decode(dataArray);
+  return { path, data };
+}
+
 export default function denoMain(): void {
   os.start("TS");
+  const { path, data } = getCompilerConfig("typescript");
```

Comment:
Would it make sense to get compiler config with `os.start` message? Seems like one less round-trip.

Reply:
It is only needed by the compiler `os.start` is shared between the compiler and the runtime.

Reply:
Yeah, I was about to write this solution is actually cleaner. Sorry for noise